v2/metadata.go

```diff
@@ -92,6 +92,18 @@ func (b *BrokerInfo) Addr() string {
 	return fmt.Sprintf("%s:%d", b.host, b.port)
 }
 
+// PartitionMetadata ontains an immutable snapshot of information for a partition
+type PartitionMetadata struct {
+	ID            int32
+	Leader        string
+	Replicas      []string
+	Isr           []string
+	HighWatermark int64
+	NewestOffset  int64
+	Paused        bool
+	LastUpdated   time.Time
+}
```

Comment:
This seems redundant with `PartitionInfo`. Can we just use that instead?

Reply:
Indeed. I should have seen that.